rules_building_block/collection_common_compressed_archived_file.toml

```diff
@@ -37,7 +37,7 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-file where event.type in ("creation", "change") and
+file where event.type in ("creation", "change") and process.executable != null and not user.id : "S-1-5-18" and
```

Comment:
I don't think we should tune this rule. The intent is to identify any file that is compressed without depending on the file extension. It should be used as contextual, which is why it is a BBR. Its need was based on identified gaps of other detection methods for compression which were more brittle

Reply:
The problem is matching on system functionality instead of user behaviors, like the firefox one represents 20% of the volume (2M alerts if I remember correctly), just related to cache files and other browser-related stuff